### PR TITLE
Adding Docker and Kubernetes support

### PR DESCRIPTION
**Pull request by nosyrbllewe**
We need to add Docker and Kubernetes support to this repository to get it able to work with the rest of project. Here we have a preliminary setup for Docker and Kubernetes support.

### CODE REVIEW

**Comment by ibnetahir1** [comment]
These files currently assume the file structure of the "take2" branch. Also, the docker is setup to use PostgreSQL while the "take2" branch currently uses SQL Server. It is planned to be changed to PostgreSQL, so in the docker files assume we are using PostgreSQL.

**Comment by ibnetahir1** [comment]
we also need to set up azure pipeline on the azure site

**Comment by ibnetahir1** [comment]
reverted the merge to keep the file changed to minimum

**Review by jpw547-zz — COMMENTED** [review]
These files seem like they should work fine.

File: .kubernetes/kubernetes_deployments.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      - name: api
+        image: fakebook.azurecr.io/fakebookauth:latest
+        ports:
+        - containerPort: 80
+        env:
+        - name: ConnectionStrings__FakebookAuth
+          value: 'Host=fakebook-auth-db;Database=postgres;Username=postgres;Password=Pass@word'
+        envFrom:
+        - secretRef:
+            name: fakebook-auth-config
```
Comment (escalonn) [inline .kubernetes/kubernetes_deployments.yaml:36]: i'll create this secret on the cluster but it'll be empty until i hear otherwise

**Review by escalonn — APPROVED** [review]

**Comment by escalonn** [comment]
ideally there'd be skeleton code in there, enough to get the pipeline passing. but fine.

**Comment by escalonn** [comment]
?? github's diff display is wrong. i did not realize that this had the entire other branch full of code in it too. that is a real pain.

**Comment by escalonn** [comment]
this is broken now. you can't simply revert a merge like was done here 20 minutes ago, it doesn't work as clean as that, because the previous commits are still in the history of the branch and that will affect any subsequent calculation of what changes are or are not in the branch.

i'm going to do some resets and force-pushes to undo both that merge and this merge.

**Comment by jpw547-zz** [comment]
would it help instead if we made a new branch off of `take2` and then open a new PR to pull into `main` instead of doing the resets and stuff? Then we can also include changes to switch it over to Postgres instead of using SqlServer?

**Pull request merged**